content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -201,6 +201,8 @@ can reach each other directly without an Istio gateway.
 
 A single network allows Istio to configure service consumers in a uniform
 way across the mesh with the ability to directly address workload instances.
+Note, however, that for a single network across multiple clusters,
```

Comment:
Definitely endpoints is true, I am not 100% sure we cannot have overlapping Service. I think @ramaraochavali may have mentioned this?

Reply:
Otherwise a call to service A can be routed to serviceB, if via VIP.

Reply:
Yes. ClusterIP has to be unique in a single network.